sklearn/ensemble/voting_classifier.py

```diff
@@ -45,6 +72,15 @@ class VotingClassifier(BaseEstimator, ClassifierMixin, TransformerMixin):
         predicted class labels (`hard` voting) or class probabilities
         before averaging (`soft` voting). Uses uniform weights if `None`.
 
+    n_jobs : int, optional (default=1)
+        The number of jobs to run in parallel for both `fit` and `predict`.
+        If -1, then the number of jobs is set to the number of cores.
+
+    backend: str, {'multiprocessing', 'threading'} (default='multiprocessing')
```

Comment:
I don't think we should allow the user to choose this. AFAIK, the threading backend is useful only if the code is written in a nogil block and the user is highly unlikely to know in which estimators that this is being done.

We should maybe just allow the default "multiprocessing" backend just like in `cross_val_score` and in other places.